hm_pyhelper/hardware_definitions.py

```diff
@@ -415,6 +415,24 @@ def is_raspberry_pi() -> bool:
         'CONTAINS_FCC_IDS': [],
         'IC_IDS': [],
         'CONTAINS_IC_IDS': []
+        },
+
+    # Linxdot CM4
+    'COMP-LINXDOTCM4': {
+        'FRIENDLY': 'Linxdot',
+        'SPIBUS': 'spidev0.0',
+        'KEY_STORAGE_BUS': '/dev/i2c-1',
+        'RESET': 17,
+        'MAC': 'wlan0',
+        'STATUS': 22,
```

Comment:
We are moving towards making `STATUS` an object so it can represent more complex lights. [See](https://github.com/NebraLtd/hm-pyhelper/blob/b48f500b407d3c2ce8651844bb8a939746847691/hm_pyhelper/hardware_definitions.py#L350)

Extra context: https://github.com/NebraLtd/hm-pyhelper/pull/103